React-JS-Projects/Basic/Password-Generator/README.md

```diff
@@ -0,0 +1,68 @@
+<h1 align='center'><b>💥 CosmoXplore : APoD 💥</b></h1>
```

Comment:
still readme is not updated!
go through readme template and apply here and change the title ok